src/components/PassengerShowcaseForm.js

```diff
@@ -364,8 +364,8 @@ const PassengerShowcaseForm = () => {
             submission.)
           </span>
         </label>
-        <label>Social media permission</label>
         <label>
+          Social media permission
           <span className={css.radioLabel}>
             <input
```

Comment:
This change makes the label "Social media permission" a part of the option _"The Cafe Car: ..."_, which makes it so that clicking on the label "Social media permission" selects that option, which may not be intended.

Reply:
Good catch, that was unintended. I made that quick edit to get rid of the linter warning (labels without an associated input are technically invalid markup). We can revert this change and I'll properly address the problem when I tackle #1550.